ports/OpenBSD/VictoriaMetrics/Makefile

```diff
@@ -15,22 +15,27 @@ MAINTAINER =		VictoriaMetrics <info@victoriametrics.com>
 # Apache License 2.0
 PERMIT_PACKAGE =	Yes
 
-WANTLIB =		c pthread
+WANTLIB =		c pthread zstd
 
 USE_GMAKE =		Yes
 
 MODULES =		lang/go
 MODGO_GOPATH =		${MODGO_WORKSPACE}
 SUBST_VARS =		VARBASE
 
+LIB_DEPENDS =		archivers/zstd
+
 do-build:
 	cd ${WRKSRC} && GOOS=openbsd ${MAKE_ENV} ${MAKE_PROGRAM} victoria-metrics-pure
 	cd ${WRKSRC} && GOOS=openbsd ${MAKE_ENV} ${MAKE_PROGRAM} vmbackup
 	cd ${WRKSRC} && GOOS=openbsd ${MAKE_ENV} ${MAKE_PROGRAM} vmrestore
+	cd ${WRKSRC} && CGO_LDFLAGS="-L${LOCALBASE}/lib -lzstd" \
```

Comment:
I think it would be better building `vmagent-pure` - then `zstd` dependency isn't needed. The same applies to `vmbackup` and `vmrestore`.

Reply:
Thank you